cmd/util/util.go

```diff
@@ -0,0 +1,43 @@
+package util
+
+import (
+  "fmt"
+  "os"
+  "os/exec"
+  "strings"
+
+  "github.com/fatih/color"
+)
+
+func CommandOutputContains(cmd []string, value string) bool {
+  return strings.Contains(GetCommandOutput(cmd), value)
+}
+
+func ContainsString(list []string, item string) bool {
+  for _, element := range(list) {
+    if item == element {
+      return true
+    }
+  }
+  return false
+}
+
+func ExitWithErrorMessage(message string) {
+  errHeaderFmt := color.New(color.Bold).Add(color.FgRed)
+  errMessageFmt := color.New(color.FgRed)
+  fmt.Println()
+  errHeaderFmt.Println("  Error")
+  errMessageFmt.Printf("  %s\n", message)
+  fmt.Println()
+  os.Exit(1)
+}
+
+func GetCommandOutput(cmd []string) string {
+  subProcess := exec.Command(cmd[0], cmd[1:]...)
+  output, err := subProcess.CombinedOutput()
+  if err != nil {
```

Comment:
We have some commands that exit with an error where we need this behavior. I can break this up so we have to explicit about the errors we want to treat as empty strings.